File: csharp/src/impl/OidHashTable.cs
namespace Volante.Impl
{
    using Volante;

    public interface OidHashTable
    {
        bool Remove(int oid);
        void Put(int oid, IPersistent obj);
        IPersistent Get(int oid);
        void Flush();
        void Invalidate();
        void SetDirty(int oid);
        void ClearDirty(int oid);
    }
}

File: java/src15/org/nachodb/impl/OidHashTable.java
package org.nachodb.impl;
import  org.nachodb.IPersistent;

public interface OidHashTable { 
    boolean     remove(int oid);
    void        put(int oid, IPersistent obj);
    IPersistent get(int oid);
    void        flush();
    void        invalidate();
    int         size();
    void        setDirty(int oid);
    void        clearDirty(int oid);
}
